internal/pkg/git/git.go

```diff
@@ -56,18 +54,16 @@ func runGitCommand(logger log.Logger, dir string, args []string) (err error, std
 	cmd := exec.Command("git", args...)
 	cmd.Dir = dir
 	cmd.Stdout = logger.DebugWriter()
-	cmd.Stderr = logger.DebugWriter()
 	cmd.Stderr = io.MultiWriter(logger.DebugWriter(), &stdErrBuffer)
 	cmd.Env = os.Environ()
 	cmd.Env = append(cmd.Env, "GIT_TERMINAL_PROMPT=0")
 	err = cmd.Run()
 	stdErr = stdErrBuffer.String()
 	exitCode = 0
 	if err != nil {
-		var exitError *exec.ExitError
-		if errors.As(err, &exitError) {
-			ws := exitError.Sys().(syscall.WaitStatus)
-			exitCode = ws.ExitStatus()
+		// nolint: errorlint
```

Comment:
![image](https://user-images.githubusercontent.com/19371734/152311651-d1a12e18-bfb8-479c-9163-6c4cfe76578b.png)

Reply:
linter dava dobre upozornenie, ... ale ked vies, ze ti to vrati prave tu chybu (a nebude wrapped do inej), tak to takto dava zmysel (vypnut linter).